react-native/react/actions/tracker.js

```diff
@@ -84,10 +86,40 @@ export function onFollowHelp (username: string): Action {
   }
 }
 
-export function onCloseFromActionBar (username: string): Action {
-  return {
-    type: Constants.onCloseFromActionBar,
-    payload: {username}
+function trackUser (username: string, state: {tracker: RootTrackerState}): Promise<boolean> {
+  const trackers = state.tracker.trackers
+  const shouldFollow = trackers[username] && trackers[username].shouldFollow
```

Comment:
make make a temp store of tracker[username] so we don't keep doing the dereference